README.md

```diff
@@ -105,6 +105,102 @@ This installation guide is for ISIS3 users interested in installing ISIS3 (3.6.0
         for Anaconda 3.4 and up - conda activate isis3
         prior to Anaconda 3.4 - source activate isis3
 
+
+### Practical Usage with other conda packages
+
+If you don't use conda for anything else on your computer, you can
+skip this section.
+
+If you use conda to install other packages, you may run into
+difficulties with adding the isis conda pacakge to those environments
+or adding other conda pacakges to the isis environment you just
+created above.  This is because the isis conda package pins a number
+of requirements that may clash with other packages.
+
+At this time, we recommend creating the isis environment as detailed
+above, and then not adding any other conda packages to it.  This
+is similar to the best practice usage of not adding any conda
+packages to your 'base' conda environment.
+
+Instead, when you need to have a conda environment with other
+packages that also needs to be able to run ISIS programs, we have
+two different options.  In both cases, we'll assume that you create
+a new environment called 'working' (but it could be named anything)
+that you want to add some conda packages to, but from which you
+also want ISIS access.
+
+The first step is to create 'working' and add whatever conda packages you want.
+
+#### Easy mode, with stacking
+
+1.  `conda activate isis3`
+
+2.  `conda activate --stack working`
+
+That's it.  Told you it was easy.
+
+This activates the isis3 environment, gets it all set up, and
+then it 'stacks' the new working environment on top of it.  To get
+out, you'll have to `conda deactivate` two times to get out of
+`working` and then out of `isis3`.
+
+
+#### Harder mode, with activation script hacking
+
+The above stacking situation may have issues if you have a particularly
+complicated set of packages or other dependencies.  The idea here is that
+the only thing you *really* need in your 'working' environment are
+the ISIS environment variables and the path to the ISIS executables.
+
+And we can do this via customizations in the conda environment's
+activate.d/ and deactivate.d/ directories.  Adding these things can
+also be done manually from the command line, but encoding them in
+the activate.d/ and deactivate.d/ scripts is handy.
+
+1.  Create your conda environment however you like, adding whatever
+packages you need.  If you were reading the directions above, you've
+already done this.
+
+2.  Locate the path to your ISIS conda environment.  Depending on
+how you installed conda, it may be in `$HOME/.anaconda3/envs/isis3/`
+or somewhere similar.  Likewise, your new 'working' environment
+will then be at `$HOME/.anaconda3/envs/working`.
+
+3.  Copy the ISIS activation and deactivation scripts to your new environment:
+
+        cd $HOME/.anaconda3/envs/
```

Comment:
Here we probably want more description about the what is going on. For example, many people will have installations in `$HOME/anaconda3` or `$HOME/miniconda3`. For the copy/paste user this will cause issues.

Reply:
D'oh!  Right, I put anaconda in a leading-dot directory because I didn't want to see it all the time, I forgot that wasn't standard.  Fixed, and tried to add some extra explanation.